service/history/workflowTaskHandlerCallbacks.go

```diff
@@ -774,7 +785,9 @@ func (handler *workflowTaskHandlerCallbacksImpl) createRecordWorkflowTaskStarted
 	// before it was started.
 	response.ScheduledEventId = workflowTask.ScheduledEventID
 	response.StartedEventId = workflowTask.StartedEventID
-	response.StickyExecutionEnabled = ms.IsStickyTaskQueueEnabled()
+	if ms.IsStickyTaskQueueEnabled() {
```

Comment:
this has the same effect as the old code? why change it?

Reply:
yes, same effect. Make 2 lines into 1. :)